main.go

```diff
@@ -30,7 +30,7 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
-	selfservicev1 "github.com/AlonaKaplan/selfserviceoverlay/api/v1"
+	selfservicev1 "github.com/AlonaKaplan/selfserviceoverlay/api/v1alpha1"
```

Comment:
selfservicev1alpha

Reply:
I'm not even sure why it is aliased.
Done